modal/cli/launch.py

```diff
@@ -16,22 +18,49 @@
 )
 
 
-def _launch_program(name: str) -> None:
+def _launch_program(name: str, args) -> None:
     contents = (Path(__file__).parent / "programs" / name).read_text()
+    contents = contents.replace("args: Any = {}", f"args: Any = {json.dumps(args)}")
 
     # TODO: This is a big hack and can break for unexpected $PATH reasons. Make an actual code path
     # for correctly setting up and running a program in the CLI.
     with tempfile.TemporaryDirectory() as tmpdir:
-        f = Path(tmpdir) / "_main.py"
+        f = Path(tmpdir) / name
         f.write_text(contents)
         subprocess.run(["modal", "run", f])
```

Comment:
this seems kind of janky – can you just launch it in the same process? we already have code for that (see `modal run`)

Reply:
Yeah, it's wrong. Will fix eventually once I work out how to programmatically use the client